src/chip/src/components/colors.js

```diff
@@ -0,0 +1,74 @@
+import * as Colors from '../../../colors';
+
+const {
+  NEUTRAL,
+  PURPLE,
+  BLUE,
+  TEAL,
+  GREEN,
+  YELLOW,
+  ORANGE,
+  RED,
+  MAGENTA,
+  VIOLET
+} = Colors;
+
+export default {
+  default: {
+    background: NEUTRAL.N00,
+    color: NEUTRAL.N600
+  },
+  blue: {
+    background: BLUE.B00,
+    color: BLUE.B400,
+    clickable: BLUE.B100
+  },
+  green: {
+    background: GREEN.G00,
+    color: GREEN.G400,
+    clickable: GREEN.G100
+  },
+  magenta: {
+    background: MAGENTA.M00,
+    color: MAGENTA.M400,
+    clickable: MAGENTA.M100
+  },
+  orange: {
+    background: ORANGE.O00,
+    color: ORANGE.O500,
+    clickable: ORANGE.O100
+  },
+  purple: {
+    background: PURPLE.P00,
+    color: PURPLE.P400,
+    clickable: PURPLE.P100
+  },
+  red: {
+    background: RED.R00,
+    color: RED.R400,
+    clickable: RED.R100
+  },
+  teal: {
+    background: TEAL.T00,
+    color: TEAL.T500,
+    clickable: TEAL.T100
+  },
+  violet: {
+    background: VIOLET.V00,
+    color: VIOLET.V400,
+    clickable: VIOLET.V100
+  },
+  yellow: {
+    background: YELLOW.Y00,
+    color: YELLOW.Y500,
+    clickable: YELLOW.Y100
+  },
+  primary: {
+    background: PURPLE.P300,
+    color: 'white'
+  },
+  secondary: {
+    background: PURPLE.P00,
+    color: PURPLE.P400
+  },
```

Comment:
`purple` and `secondary` look identical. Why does `secondary` need to exist?

<img width="350" alt="Screen Shot 2021-03-22 at 12 51 15 PM" src="https://user-images.githubusercontent.com/440299/112049767-4c8bd400-8b0d-11eb-93b1-93264d2d776d.png">

Reply:
secondary is default mui based on our theme. I can remove either.

Reply:
I'm removing color purple